### PR TITLE
Added docs issue control template for new feedback system

### PR DESCRIPTION
**Pull request by mikefrobbins**
## Description

Added issue control template for the new automated docs feedback system.

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by mikefrobbins** [comment]
This issue template is required for the new docs feedback system. It must remain at the bottom of the list of issue templates and separate from the existing docs issue template that already exists in this repo.

File: .github/ISSUE_TEMPLATE/6-AZ-DOC-FEEDBACK-CONTROL.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+name: "📄 Feedback control template for docs"
+title: "Feedback"
+description: >-
+  ⛔ This template is intended for use by the feedback control on the bottom of every article page on the docs site. If you aren't using the feedback control, choose one of the other templates.⛔
+labels:
+  - "needs-triage"
```
Comment (sdwheeler) [inline .github/ISSUE_TEMPLATE/6-AZ-DOC-FEEDBACK-CONTROL.yml:6]: Might want to include the "Doc - Reference" label as well.
```suggestion
  - "needs-triage,Doc - Reference"
```

**Review by sdwheeler — APPROVED** [review]
Added an optional suggestion.

File: .github/ISSUE_TEMPLATE/6-AZ-DOC-FEEDBACK-CONTROL.yml (hunk 1 above)
Comment (mikefrobbins) [reply .github/ISSUE_TEMPLATE/6-AZ-DOC-FEEDBACK-CONTROL.yml:6]: The labels match the existing docs issue template.

**Comment by mikefrobbins** [comment]
@VeryEarly Please merge this PR unless there is any feedback or questions. - Thank you

**Comment by isra-fel** [comment]
We discussed this offline and decided to combine the new issue template with the existing [about documentation issues and questions](https://github.com/Azure/azure-powershell/issues/new?assignees=&labels=needs-triage&projects=&template=4-AZ-DOC-BUG.yml&title=%5BDoc%5D%3A+) because seeing two categories about docs may confuse the customers.

That means we may need to redesign this template. Maybe make the pre-populating mandatory fields optional.

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**